report/04_implementation/01_flutter_application.md

```diff
@@ -50,3 +176,10 @@ are clearly separated from each other so that errors or changes only have to
 be carried out once and not several times. In addition, the individual areas
 should be easy to find by other programmers.
 
+### Testing
+
+Testing within soTired was considered as important as every other part of the 
```

Comment:
```suggestion
Testing within soTired was considered as very important like every other part of the 
```

Reply:
You can also use 'considered as as important as every ...', but that sounds a little bit weird. For me in the orginal statement one 'as' is missing..

Reply:
`as ... as` is commonly used to compare things. Therefore, I think I have to insist on leaving this untouched 😅